presto-product-tests/bin/run_on_docker.sh

```diff
@@ -207,7 +207,7 @@ fi
 trap terminate INT TERM EXIT
 
 # start external services
-EXTERNAL_SERVICES="hadoop-master mysql postgres cassandra"
+EXTERNAL_SERVICES="hadoop-master mysql postgres cassandra sqlserver"
```

Comment:
We're running quite a lot of additional services for the product tests, even if they aren't used. Does it pass on travis? We should think how to only run the necessary services. For this PR it's fine though as long as the build is stable